vivarium/core/store.py

```diff
@@ -582,47 +582,51 @@ def _apply_config(self, config, source=None):
             config = {}  # config needs to be a dict
 
         # remove _output special key. This is used only by schema_topology.
-        config = without(config, '_output')
+        config = config.copy()
+        config.pop('_output', None)
 
         if '*' in config:
-            self._apply_subschema_config(config['*'])
-            config = without(config, '*')
+            config = config.copy()
```

Comment:
Do we need to copy `config` every time here? shouldn't the `config = config.copy()` be sufficient to make sure we are not mutating the original config?

Reply:
Great point! That should make it even faster than before.